bounties/maven/kindeditor/1/README.md

```diff
@@ -0,0 +1,5 @@
+# Overview
+
+`kindeditor` is a lightweight, Open Source(LGPL), cross browser, web based WYSIWYG HTML editor.
```

Comment:
The issue is still opened, but in the title is specified it affects only versions <= 4.1.1 and that was the version in early Nov 2018, while some changes have been committed on end-Dec 2019.

Also, in the `master` branch the vulnerable file has been renamed with a `txt` extension: https://github.com/kindsoft/kindeditor/blob/master/php/upload_json.php.txt (not sure if it has been done to avoid this attack or simply is no more needed).

That said, it seems still present so I would `accept` it and see if it's possible fix the issue, even if I think it's more a `excepted behavior` since the root is `/`  :+1: